src/views/edit/PlateContainer.tsx

```diff
@@ -337,12 +337,7 @@ export default observer(
     );
 
     return (
-      <Plate
-        initialValue={value as any}
-        onChange={setValue}
-        plugins={plugins}
-        readOnly={saving}
```

Comment:
I'm not sure why this line was here, and also unsure why this didn't trigger noticeable focus loss before; would have been less noticeable with the prior settings, but should have still been noticeable...